src/chrome/content/rules/Rambler.ru.xml

```diff
@@ -1,74 +1,38 @@
-Fetch error: http://krasnodar-realty.rambler.ru/ => https://krasnodar-realty.rambler.ru/: (6, 'Could not resolve host: krasnodar-realty.rambler.ru')
-Fetch error: http://magas-realty.rambler.ru/ => https://magas-realty.rambler.ru/: (6, 'Could not resolve host: magas-realty.rambler.ru')
-Fetch error: http://maykop-realty.rambler.ru/ => https://maykop-realty.rambler.ru/: (6, 'Could not resolve host: maykop-realty.rambler.ru')
-Fetch error: http://nalchik-realty.rambler.ru/ => https://nalchik-realty.rambler.ru/: (6, 'Could not resolve host: nalchik-realty.rambler.ru')
-Fetch error: http://naryan-mar-realty.rambler.ru/ => https://naryan-mar-realty.rambler.ru/: (6, 'Could not resolve host: naryan-mar-realty.rambler.ru')
-Fetch error: http://nn-realty.rambler.ru/ => https://nn-realty.rambler.ru/: (6, 'Could not resolve host: nn-realty.rambler.ru')
-Fetch error: http://orenburg-realty.rambler.ru/ => https://orenburg-realty.rambler.ru/: (6, 'Could not resolve host: orenburg-realty.rambler.ru')
-Fetch error: http://penza-realty.rambler.ru/ => https://penza-realty.rambler.ru/: (6, 'Could not resolve host: penza-realty.rambler.ru')
-Fetch error: http://petrozavodsk-realty.rambler.ru/ => https://petrozavodsk-realty.rambler.ru/: (6, 'Could not resolve host: petrozavodsk-realty.rambler.ru')
-Fetch error: http://rostov-na-donu-realty.rambler.ru/ => https://rostov-na-donu-realty.rambler.ru/: (6, 'Could not resolve host: rostov-na-donu-realty.rambler.ru')
-Fetch error: http://saransk-realty.rambler.ru/ => https://saransk-realty.rambler.ru/: (6, 'Could not resolve host: saransk-realty.rambler.ru')
-Fetch error: http://simferopol-realty.rambler.ru/ => https://simferopol-realty.rambler.ru/: (6, 'Could not resolve host: simferopol-realty.rambler.ru')
-Fetch error: http://spb-realty.rambler.ru/ => https://spb-realty.rambler.ru/: (6, 'Could not resolve host: spb-realty.rambler.ru')
-Fetch error: http://syktyvkar-realty.rambler.ru/ => https://syktyvkar-realty.rambler.ru/: (6, 'Could not resolve host: syktyvkar-realty.rambler.ru')
-Fetch error: http://tver-realty.rambler.ru/ => https://tver-realty.rambler.ru/: (6, 'Could not resolve host: tver-realty.rambler.ru')
-Fetch error: http://ulyanovsk-realty.rambler.ru/ => https://ulyanovsk-realty.rambler.ru/: (6, 'Could not resolve host: ulyanovsk-realty.rambler.ru')
-Fetch error: http://vladikavkaz-realty.rambler.ru/ => https://vladikavkaz-realty.rambler.ru/: (6, 'Could not resolve host: vladikavkaz-realty.rambler.ru')
-Fetch error: http://vladimir-realty.rambler.ru/ => https://vladimir-realty.rambler.ru/: (6, 'Could not resolve host: vladimir-realty.rambler.ru')
-Fetch error: http://yoshkar-ola-realty.rambler.ru/ => https://yoshkar-ola-realty.rambler.ru/: (6, 'Could not resolve host: yoshkar-ola-realty.rambler.ru')
-
-403 Forbidden:
-	cnt.rambler.ru
-	counter.rambler.ru
-	scounter.rambler.ru
-	tools.rambler.ru
+	403 Forbidden:
+		cnt.rambler.ru
+		counter.rambler.ru
+		scounter.rambler.ru
```

Comment:
You just removed them and their working test urls. Please revert this and re-activate them. You may add an exclusion for `$` that Travis only tests the custom test url.